bin/bb/cmd/migrate.go

```diff
@@ -19,30 +20,19 @@ import (
 
 func newMigrateCmd() *cobra.Command {
 	var (
-		databaseType string
-		username     string
-		password     string
-		hostname     string
-		port         string
-		database     string
-		fileList     []string
-		commandList  []string
-		description  string
-		issueID      string
-
-		// SSL flags.
-		sslCA   string // server-ca.pem
-		sslCert string // client-cert.pem
-		sslKey  string // client-key.pem
+		dsn         string
+		fileList    []string
+		commandList []string
+		description string
+		issueID     string
 	)
 	migrateCmd := &cobra.Command{
 		Use:   "migrate",
 		Short: "Migrate the database schema",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			tlsCfg := db.TLSConfig{
-				SslCA:   sslCA,
-				SslCert: sslCert,
-				SslKey:  sslKey,
+			u, err := dburl.Parse(dsn)
+			if err != nil {
+				return fmt.Errorf("failed to parse dsn, got error: %w", err)
```

Comment:
if this error will result in printing out the --help, then we can just add instruction there, otherwise, we need to duplicate the instruction for the format.

Reply:
yes, when runE() returns an error, cobra will print out --help